arrus/core/api/devices/us4r/Us4R.h

```diff
@@ -298,6 +298,9 @@ class Us4R: public DeviceWithComponents, public Ultrasound {
      */
     virtual void setAfe(uint8_t reg, uint16_t val) = 0;
 
+
+	virtual void hvpsSetVoltage(float voltage) = 0;
```

Comment:
Why do we need one more function to set voltage?

Reply:
Used forr debug purposes, now removed